api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -265,6 +268,52 @@ public ResponseEntity<CohortReviewListResponse> getCohortReviewsInWorkspace(
                 cohortReviewService.getRequiredWithCohortReviews(workspaceNamespace, workspaceId)));
   }
 
+  @Override
+  public ResponseEntity<DemoChartInfoListResponse> findCohortReviewDemoChartInfo(
+      String workspaceNamespace,
+      String workspaceId,
+      Long cohortReviewId,
+      String genderOrSex,
+      String age) {
+    workspaceAuthService.getWorkspaceEnforceAccessLevelAndSetCdrVersion(
+        workspaceNamespace, workspaceId, WorkspaceAccessLevel.READER);
+
+    Set<Long> participantIds = cohortReviewService.findParticipantIdsByCohortReview(cohortReviewId);
```

Comment:
We need to keep an eye on this method and make sure it's not slow. Cohort reviews can contain up to 10,000 participants. This method is reading the DbParticipantCohortStatus object  which is a bit heavier than just a participant id. If this is slow we may need a new dao call that returns a list participant ids instead of a list of DbParticipantCohortStatus.

Reply:
Right. I will add new SQL to get just the Ids.